packages/services/server/src/supertokens/oidc-provider.ts

```diff
@@ -29,21 +29,24 @@ export const getOIDCSuperTokensOverrides = (): ThirdPartEmailPasswordTypeInput['
 
 export type BroadcastOIDCIntegrationLog = (oidcId: string, message: string) => void;
 
+function getLoggerFromUserContext(userContext: unknown): FastifyBaseLogger {
+  return (userContext as any)._default.request.request.log;
```

Comment:
This one is a bit messy, but I guess no other way to access it?

Reply:
SuperTokens :)